native/src/main/java/io/ballerina/stdlib/http/api/HttpDispatcher.java

```diff
@@ -436,6 +449,51 @@ static BObject createRequestContext(HttpCarbonMessage httpCarbonMessage) {
         return requestContext;
     }
 
+    private static void addJwtValuesToRequestContext(Runtime runtime, BObject requestContext, String authHeader) {
+        Object decodedJwt = invokeJwtDecode(runtime, authHeader);
+        if (Objects.nonNull(decodedJwt)) {
+            BMap requestCtxMembers = requestContext.getMapValue(REQUEST_CTX_MEMBERS);
+            requestCtxMembers.put(JWT_INFORMATION, decodedJwt);
+        }
+    }
+
+    private static Object invokeJwtDecode(Runtime runtime, String authHeader) {
+        final Object[] jwtInformation = new Object[1];
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        Callback decodeCallback = new Callback() {
+            @Override
+            public void notifySuccess(Object result) {
+                if (!(result instanceof Exception)) {
```

Comment:
Here you are silently ignoring the error right? So the user is not going to get the error you are returning from the ballerina jwt decode function right?

Reply:
Yeah. Ignored the error because the header can also be oauth2 and then providing an error saying we are expecting jwt is confusing.
But if the user actually sent a jwt and there is a jwt decoding error, user wont see it and he will only see `"no member found for key: JWT_INFORMATION"`. This is also a little bit misleading as well.
Will record this in the issue.